src/core/resources/assets/userAssets.ts

```diff
@@ -219,7 +220,9 @@ export async function parseUserAssets({
   chainIds: ChainId[];
   currency: SupportedCurrencyKey;
 }) {
-  const { connectedToHardhat } = connectedToHardhatStore.getState();
+  const { connectedToHardhat, connectedToHardhatOp } =
+    useConnectedToHardhatStore.getState();
```

Comment:
but for ex here you use `.getState()` instead of the hook because it's a `parseUserAssets` function, it doesn't need to react to these changes and would probably break the rule of hooks because its called from react query and not directly from a component/other hook

Reply:
not my best explanation happy to elaborate on slack if you want lol